src/platform/audio/milesaudiomanager.cpp

```diff
@@ -1985,7 +1985,7 @@ void MilesAudioManager::Adjust_Playing_Volume(PlayingAudio *audio)
             }
 
             AIL_set_stream_volume_pan(audio->miles.stream, vol, pan);
-        }
+        } break;
         default:
```

Comment:
Is this the convention break after the scope? I've worked on different codebases and everyone seems to have a different idea.

Reply:
I am not aware about convention. Either way is fine.

Reply:
Coding style says this is the convention.